core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -339,6 +349,20 @@ public boolean shutdownWorkerGroupOnStop() {
         return shutdownWorkerGroupOnStop;
     }
 
+    /**
+     * Returns the {@link ChannelOption}s and their values of {@link Server}'s server sockets.
+     */
+    public Map<ChannelOption<?>, Object> serverChannelOptions() {
+        return Collections.unmodifiableMap(serverChannelOptions);
+    }
+
+    /**
+     * Returns the {@link ChannelOption}s and their values of {@link Server}'s server child sockets.
```

Comment:
not sure `child sockets` is correct word to used or not, please give me advices.

Reply:
How about: the sockets accepted by the {@link Server}